Tiltfile

```diff
@@ -61,11 +62,54 @@ def deploy_capi():
             if kb_extra_args:
                 patch_args_with_extra_args("capi-kubeadm-bootstrap-system", "capi-kubeadm-bootstrap-controller-manager", kb_extra_args)
 
+## This should have the same versions as the Dockerfile
+if settings.get("local_mode"):
+    tilt_dockerfile_header_cso = """
+    FROM ghcr.io/sovereigncloudstack/cso:v0.1.0-alpha.2 as builder
+
+    FROM docker.io/library/alpine:3.18.0 as tilt
+    WORKDIR /
+    COPY --from=builder /usr/local/bin/helm /usr/local/bin/helm
+    COPY --from=builder /manager /manager
+    COPY .release/ /tmp/downloads/cluster-stacks/
+    """
+else:
+    tilt_dockerfile_header_cso = """
+    FROM ghcr.io/sovereigncloudstack/cso:v0.1.0-alpha.2 as builder
+
+    FROM docker.io/library/alpine:3.18.0 as tilt
+    WORKDIR /
+    COPY --from=builder /usr/local/bin/helm /usr/local/bin/helm
+    COPY --from=builder /manager /manager
+    """
+    
+
 def deploy_cso():
+    # yaml = str(kustomizesub("./hack/observability")) # build an observable kind deployment by default
     version = settings.get("cso_version")
     cso_uri = "https://github.com/sovereignCloudStack/cluster-stack-operator/releases/download/{}/cso-infrastructure-components.yaml".format(version)
     cmd = "curl -sSL {} | {} | kubectl apply -f -".format(cso_uri, envsubst_cmd)
-    local(cmd, quiet = True)
+    yaml_cso = str(local(cmd, quiet = True))
```

Comment:
```suggestion
    local(cmd, quiet = True)
```

Reply:
Yes, we know that and I think it is in the readme too. I don't have much experience with Tilt, so if you know how to make the tilt wait for the CRDs to be applied first and then apply `cspo-template.yaml` with filled env variables, that would be great.

Reply:
@kranurag7 please see #84